lib/pbench/server/database/models/tracker.py

```diff
@@ -355,6 +359,33 @@ def validate_state(self, key, value):
             raise DatasetBadParameterType(value, States)
         return value
 
+    @validates("owner")
+    def validate_owner(self, key, value):
```

Comment:
(I'm really looking forward to seeing what you come up with for hints...I'm hoping that you don't punt and use `Any` for the return value....  :-) )

Reply:
It's awkward, because `value` is flexible; after all, we're here to validate it as an unknown. But if we return anything, it's a `User`. I'm debating, though, whether it's `value: str` or `value: Any`...

The thing is, I feel a bit weird adding type hints just to this new code when they're missing from most of the rest of the APIs; but I don't want to add them all, either.

Reply:
Oh, I'm pretty sure that `value` is an `Any`.  It's the _return_ value that I want to see you describe!  (I assume it's a union of `User` and `str`, but I'm waiting to see what you come up with.)

As for adding type hints, we should not skip them for new code, and we'll get to the existing code when it seems appropriate.